server/src/main/java/io/druid/server/coordination/ZkCoordinator.java

```diff
@@ -86,8 +88,10 @@ public void loadLocalCache()
     }
```

Comment:
It looks like `loadLocalCache` is called from `io.druid.server.coordination.BaseZkCoordinator#start` as part of the normal workflow. The block immediately prior to this comment checks for existence of the info dir and returns if it nothing exists at the path and a directory cannot be created. If nothing exists and mkdirs fails, this method silently returns. It makes sense that it wouldn't try to load up cache information, but the lack of any sort of logging as to why nothing was attempted to load was surprising.